grpc/client.go

```diff
@@ -39,17 +39,17 @@ func ClientSetup(c *cmd.GRPCClientConfig, tlsConfig *tls.Config, statsRegistry p
 		return nil, err
 	}
 
-	ci := clientInterceptor{c.Timeout.Duration, metrics, clk}
+	mi := clientMetadataInterceptor{c.Timeout.Duration, metrics, clk}
```

Comment:
Nit, here and below, I think this should probably be:
```suggestion
	cmi := clientMetadataInterceptor{c.Timeout.Duration, metrics, clk}
```